app/src/bcsc-theme/features/settings/Settings.tsx

```diff
@@ -142,28 +164,37 @@ const Settings: React.FC<SettingsScreenProps> = ({ navigation }: SettingsScreenP
               <SettingsActionCard title={t('BCSCSettings.Notifications')} onPress={onPressActionTodo} />
 
               <SettingsActionCard title={t('BCSCSettings.ForgetPairings')} onPress={onPressForgetAllPairings} />
-            </View>
-          </>
-        ) : null}
 
-        <ThemedText variant={'bold'} style={styles.sectionHeader}>
-          {t('BCSCSettings.HeaderB')}
-        </ThemedText>
+              {store.preferences.developerModeEnabled && (
+                <SettingsActionCard title={t('Developer.DeveloperMode')} onPress={onPressDeveloperOptions} />
+              )}
+            </>
+          ) : null}
+
+          <ThemedText variant={'bold'} style={styles.sectionHeader}>
+            {t('BCSCSettings.HeaderB')}
+          </ThemedText>
 
-        <View style={styles.sectionContainer}>
           <SettingsActionCard title={t('BCSCSettings.Help')} onPress={onPressHelp} />
           <SettingsActionCard title={t('BCSCSettings.Privacy')} onPress={onPressPrivacy} />
           <SettingsActionCard title={t('BCSCSettings.ContactUs')} onPress={onPressContactUs} />
           <SettingsActionCard title={t('BCSCSettings.Feedback')} onPress={onPressFeedback} />
           <SettingsActionCard title={t('BCSCSettings.Accessibility')} onPress={onPressAccessibility} />
           <SettingsActionCard title={t('BCSCSettings.TermsOfUse')} onPress={onPressTermsOfUse} />
-        </View>
-
-        {/* TODO (MD): Add developer options */}
 
-        <View style={styles.versionContainer}>
-          <ThemedText variant="labelSubtitle">BC Services Card</ThemedText>
-          <ThemedText variant="labelSubtitle">{`Version ${getVersion()} (${getBuildNumber()})`}</ThemedText>
+          {/* TODO (MD): Add developer options */}
+
+          <View style={styles.versionContainer}>
+            <TouchableWithoutFeedback
+              onPress={incrementDeveloperMenuCounter}
+              disabled={store.preferences.developerModeEnabled}
+            >
+              <View>
+                <ThemedText variant="labelSubtitle">BC Services Card</ThemedText>
```

Comment:
nit: This is actually an artifact from myself I think...
Could you do me a favour and grab this value from the translations file?